packages/network/libssh/package.mk

```diff
@@ -2,12 +2,12 @@
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
 
 PKG_NAME="libssh"
-PKG_VERSION="0.7.5"
-PKG_SHA256="6f7d5b9f3089164eec7408d35b99194b17352cce436a4a95485b511bce2aab4b"
+PKG_VERSION="0.8.1"
+PKG_SHA256="d17f1267b4a5e46c0fbe66d39a3e702b8cefe788928f2eb6e339a18bb00b1924"
 PKG_ARCH="any"
 PKG_LICENSE="OpenSource"
```

Comment:
could you pls change it to `PKG_LICENSE="BSD"`

Reply:
You mean LGPL as mentioned at https://www.libssh.org/features/ ?